modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/DefaultValue.java

```diff
@@ -166,7 +166,7 @@ public boolean equals(Object o) {
 
             ConstantValue that = (ConstantValue) o;
 
-            return Objects.equals(value, that.value);
+            return Objects.deepEquals(value, that.value);
```

Comment:
what's the reason for `deepEquals` here? As far as Im concerned, `deepEquals` makes sense for arrays but here we deal with `DefaultValue`. Do I miss something?